controllers/index.js

```diff
@@ -30,6 +30,14 @@ exports.index = (req, res) => {
 
                 const hacktoberFestLabels = _.filter(event.labels, label => label.name.toLowerCase() === 'hacktoberfest');
 
+
+                //ignore pull requests on own repos
+                const repoOwner = repo.replace('https://github.com/', '').split('/')[0];
+
+                if (username.toUpperCase() === repoOwner.toUpperCase()) {
```

Comment:
Every cycle in `_.each` you call `username.toUpperCase`. It's better to call it only once before `_.each`.